src/OctoshiftCLI.Tests/ado2gh/Commands/ShareServiceConnectionCommandTests.cs

```diff
@@ -9,7 +9,7 @@ namespace OctoshiftCLI.Tests.AdoToGithub.Commands
 {
     public class ShareServiceConnectionCommandTests
     {
-        private readonly Mock<AdoApi> _mockAdoApi = TestHelpers.CreateMock<AdoApi>();
+        private readonly Mock<AdoClient> _mockAdoClient = TestHelpers.CreateMock<AdoClient>();
```

Comment:
In the command tests we shouldn't need to mock the client. You added the `AdoApi#ConatinsServiceConnection` method and all you need to do is to set it up properly in tests to return `false` for happy paths and `true` for sad paths. 
Now to test the new method (`ContainsServiceConnection`) in the `AdoApi`  itself you need to add a test in the `AdoApiTests` and in there you setup the client for that test to return the `serviceConnection` payload.